src/cards/gist-card.js

```diff
@@ -118,7 +119,7 @@ const renderGistCard = (gistData, options = {}) => {
         ? `${header.slice(0, HEADER_MAX_LENGTH)}...`
         : header,
     titlePrefixIcon: icons.gist,
-    width: CARD_DEFAULT_WIDTH,
+    width: card_width ? card_width : CARD_DEFAULT_WIDTH,
```

Comment:
Some logic is needed to set a minimum card width to prevent the card from becoming too small. See 

https://github.com/anuraghazra/github-readme-stats/blob/9c6eb2286284a44ea3ba983ab4d2d2f8a8c2203e/src/cards/top-languages-card.js#L754-L760